.npmignore

```diff
@@ -9,4 +9,5 @@ win
 0x-logo.png
 *.gif
 *.0x
-todo
\ No newline at end of file
+todo
+.vscode
```

Comment:
Can the file end with `\n` given the opportunity? This will make future diffs cleaner.